src/platform/mod.rs

```diff
@@ -7,40 +7,50 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-mod os {
-    #[cfg(any(feature = "force-inprocess", not(target_os = "macos")))]
-    struct Incrementor {
-        last_value: u64,
-    }
+#[cfg(any(feature = "force-inprocess", not(target_os = "macos")))]
+struct Incrementor {
+    last_value: u64,
+}
 
-    #[cfg(any(feature = "force-inprocess", not(target_os = "macos")))]
-    impl Incrementor {
-        fn new() -> Incrementor {
-            Incrementor {
-                last_value: 0
-            }
+#[cfg(any(feature = "force-inprocess", not(target_os = "macos")))]
+impl Incrementor {
+    fn new() -> Incrementor {
+        Incrementor {
+            last_value: 0
         }
+    }
 
-        fn increment(&mut self) -> u64 {
-            self.last_value += 1;
-            self.last_value
-        }
+    fn increment(&mut self) -> u64 {
+        self.last_value += 1;
+        self.last_value
     }
+}
 
-    #[cfg(all(not(feature = "force-inprocess"), any(target_os = "linux",
-                                                    target_os = "freebsd")))]
-    include!("unix/mod.rs");
+#[cfg(all(not(feature = "force-inprocess"), any(target_os = "linux",
+                                                target_os = "freebsd")))]
+mod unix;
```

Comment:
As I pointed out in my original review, this change is really unrelated to the actual Windows implementation, and as such belongs in a separate PR.

I don't want to argue it though; so instead I just submitted it as a separate PR myself: https://github.com/servo/ipc-channel/pull/132 -- feel free to comment :-)